lib/app_web/controllers/auth_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule AppWeb.Auth do
+  use AppWeb, :controller
+  # use AppWeb, :router
+  import Plug.Conn
+  # alias App.Repo
+
+  def init(opts), do: opts
+
+  def call(conn, _opts) do
+    person_id = get_session(conn, :person_id)
+    cond do
+      person = conn.assigns[:current_person] ->
+        assign(conn, :current_person, person)
+
+      person = person_id && App.Ctx.get_person!(person_id) ->
```

Comment:
`App.Ctx.get_person!(person_id)` appears to fail _ugly_ if there are _zero_ people in the DB.
See: https://github.com/dwyl/app-mvp-phoenix/pull/24#issuecomment-564590702

Reply:
:worried: yes I think we need to add a `case ... do` instead

Reply:
Resolved by deleting cookies in browser. ✅